src/link/link.directive.ts

```diff
@@ -0,0 +1,29 @@
+import {
+	Directive,
+	HostBinding,
+	Input
+} from "@angular/core";
+
+/**
+ * A convinence directive for applying styling to a link.
+ *
+ * Example:
+ *
+ * ```hmtl
+ * <a href="#" ibmLink>A link</a>
+ * ```
+ *
+ * See the [vanilla carbon docs](http://www.carbondesignsystem.com/components/link/code) for more detail.
+ */
+@Directive({
+	selector: "[ibmLink]"
+})
+
+export class Link {
+	@HostBinding("class.bx--link") baseClass = true;
+	/**
+	 * Set to `true` to disable link.
+	 * @memberof Link
+	 */
+	@HostBinding("attr.aria-disabled") @Input() disabled = false;
```

Comment:
Per carbon docs, `disabled` should also make the link not tabable, setting `tabindex="-1"`

This can be done by making this a [`get`ter `set`ter combo](https://www.typescriptlang.org/docs/handbook/classes.html#accessors), where `set`ter will set a private `_disabled` field and set tabindex to -1 if value is true and to null if value is false.
To be able to set tabindex, you may need `@HostBinding("attr.tabindex") tabindex;`